vmupdate/vmupdate.py

```diff
@@ -50,14 +59,26 @@ def parse_args(args):
     group.add_argument('--all', action='store_true',
                        help='Target all non-disposable VMs (TemplateVMs and '
                             'AppVMs)')
-
+    group.add_argument('--n', action='store',
```

Comment:
`--n` looks weird, I liked `--smart` more, but maybe @marmarta has some better idea?

Reply:
yes I also like `qubes-vm-update --smart` but `qubes-vm-update --smart 5` doesn't make sense to me. For now smart behavior is default so `qubes-vm-update` will do what we want, so first option is out. 
I'm open to proposition for a parameter name "how long without checking for updates is unacceptable"

Reply:
`--smart-time`, `--check-older`?

Reply:
Also chatGPT proposed '--update-if-stale N', and frankly it sounds like a good idea